internal/makefile/makefile.go

```diff
@@ -351,12 +352,15 @@ endif
 			// add target to run shellcheck
 			ignorePathArgs := ""
 			for _, path := range cfg.ShellCheck.AllIgnorePaths(cfg.Golang) {
+				if strings.HasPrefix(path, "/") {
+					logg.Fatal("ShellCheck ignore paths must not start with a slash, got: %s", path)
+				}
+
 				// https://github.com/ludeeus/action-shellcheck/blob/master/action.yaml#L120-L124
 				if !strings.HasPrefix(path, "./") {
-					ignorePathArgs += fmt.Sprintf(" ! -path '*./%s/*'", path)
-					ignorePathArgs += fmt.Sprintf(" ! -path '*/%s/*'", path)
+					ignorePathArgs += fmt.Sprintf(" \\( -path '*./%s/*' -prune \\) -o", path)
```

Comment:
This extra dot looks weird.
```suggestion
					ignorePathArgs += fmt.Sprintf(" \\( -path '*/%s/*' -prune \\) -o", path)
```